README.md

```diff
@@ -2,12 +2,14 @@
 
 **MSAL Go is a new addition to the MSAL family of libraries, not yet in public preview, that has been made available to gauge customer interest and to gather early feedback from the community. We welcome all contributors (see [CONTRIBUTING.md](https://github.com/AzureAD/microsoft-authentication-library-for-go/blob/dev/CONTRIBUTING.md)) to help us grow our list of supported MSAL SDKs.**
 
-The MSAL library for Go is part of the [Microsoft identity platform for developers](https://aka.ms/aaddevv2) (formerly named Azure AD) v2.0. It enables you to acquire security tokens to call protected APIs. It uses industry standard OAuth2 and OpenID Connect.
```

Comment:
>SAL library for Go is part of the [Microso [](start = 5, length = 42)

This is what we have for all our repo's not sure this should be removed?

Reply:
Good catch, I was referring to MSAL [Java](https://github.com/AzureAD/microsoft-authentication-library-for-java#microsoft-authentication-library-msal-for-javal) and [Python](https://github.com/AzureAD/microsoft-authentication-library-for-python#microsoft-authentication-library-msal-for-python). So followed a similar format. Just looked at MSAL .net which has it in this format. Adding it back with the link in the next commit